### PR TITLE
add testlink chart

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

**Comment by prydonius** [comment]
@k8s-bot ok to test

File: stable/testlink/values.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+## Default Language
+## ref: https://github.com/bitnami/bitnami-docker-testlink#environment-variables
+##
+testlinkLanguage: en_US
+
+## SMTP mail delivery configuration
+## ref: https://github.com/bitnami/bitnami-docker-testlink#smtp-configuration
+##
+# smtpEnable:
```
Comment (viglesiasce) [inline stable/testlink/values.yaml]: Can you add examples of sane values in these parameters?

**Review by viglesiasce — CHANGES_REQUESTED** [review]

**Comment by sameersbn** [comment]
Thanks for the review @viglesiasce.

I realized that the bitnami/testlink image does not currently support configuring SMTP parameter. Therefore I've gone ahead and removed these parameters from the chart. 

Will add these parameters in future updates of the chart when they are properly supported in the image.

**Comment by sameersbn** [comment]
chart image updated to `bitnami/testlink:1.9.14-r4` which contains fixes for SMTP configuration.

**Review by viglesiasce — APPROVED** [review]

**Pull request merged**